Assets/HoloToolkit/Input/Scripts/InputHandlers/IGamePadHandler.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine.EventSystems;
+
+namespace HoloToolkit.Unity.InputModule
+{
+    public interface IGamePadHandler : IEventSystemHandler
+    {
+        void OnGamePadDetected(GamePadEventData eventData);
```

Comment:
Do you think it's more useful making gamepad input separate events, or to combine them with the motion controller events? They share a set of input axes, and source detected/lost could be handled similarly.

Reply:
I kept it as a singular event to reduce overhead.

I'm already thinking about a plan to make sure sure this plays well with the motion controllers in the dev branch.  I'll be sure to request your feedback when I submit.